Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -33,15 +33,18 @@ class BTPayPalNativeTokenizationClient {
         ) { body, _, error in
             guard let json = body, error == nil else {
                 let underlyingError = error ?? BTPayPalNativeError.invalidJSONResponse
+                self.apiClient.sendAnalyticsEvent("ios.paypal-native.tokenize.failed")
```

Comment:
Similar comment to above. If the other events are granular, this one also might as well be?